config/initializers/devise.rb

```diff
@@ -272,7 +272,13 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-  config.omniauth :google_oauth2, ENV["GOOGLE_OAUTH_CLIENT_ID"], ENV["GOOGLE_OAUTH_CLIENT_SECRET"], scope: "email"
+  # the 'redirect_uri:' is here so that Devise always knowns about it.
+
+  # provider_ignores_state: true :-
+  # This tells the OAuth provider to ignore the `state` parameter during the OAuth flow. The `state` parameter is typically used as a security measure to prevent cross-site request forgery (CSRF) attacks.
```

Comment:
Here is a description of the sort of attack that is possible if you don't use the `state` param as an anti-CSRF measure - https://stackoverflow.com/a/35988614. I don't think it applies to us. Because it's about how malicious person A would get an access code for their own Google account and sneakily get person B to use that access token to authenticate to our site. But this only works if malicious person A controls a Google account that has admin access to our site, and tricking person B into logging into our site as person A doesn't really get them any benefit.